app/assets/stylesheets/application.scss

```diff
@@ -17,21 +17,28 @@
 @import "bootstrap";
 
 body {
-  background-color: #e5e8e0;
+  background-color: #e0e0e0;
```

Comment:
Color literals like `#e0e0e0` should only be used in variable declarations; they should be referred to via variable everywhere else.